common/script_values/babylon_values.txt

```diff
@@ -748,3 +748,435 @@ babylon_player_construction_bonus_multiplier = {
 		}
 	}
 }
+
+babylon_num_of_great_powers = {
+	value = 0
+	every_country = {
+		limit = {
+			country_rank = rank_value:great_power
+		}
+		add = 1
+	}
+}
+
+babylon_num_of_major_powers = {
+	value = 0
+	every_country = {
+		limit = {
+			country_rank = rank_value:major_power
+		}
+		add = 1
+	}
+}
+
+babylon_num_of_congress_attendees = {
+	value = 0
+	every_country = {
+		limit = {
+			is_congress_attendee = yes
+		}
+		add = 1
+	}
+}
+
+total_prestige_of_voting_conference_members = {
+	value = 0
+	every_country = {
+		limit = {
+			OR = {
+				has_variable = voting_conference_member
+				is_target_in_global_variable_list = {
+					name = current_congress_members_on_stance_1
+					target = this
+				}
+				is_target_in_global_variable_list = {
+					name = current_congress_members_on_stance_2
+					target = this
+				}
+				is_target_in_global_variable_list = {
+					name = current_congress_members_on_stance_3
+					target = this
+				}
+			}
+		}
+		add = prestige
+	}
+}
+
+conference_supporters_prestige_value = {
+	value = 0
+	every_country = {
+		limit = {
+			is_target_in_global_variable_list = {
+				name = current_congress_members_on_stance_1
+				target = this
+			}
+		}
+		add = prestige
+	}
+}
+
+conference_neutral_prestige_value = {
+	value = 0
+	every_country = {
+		limit = {
+			is_target_in_global_variable_list = {
+				name = current_congress_members_on_stance_2
+				target = this
+			}
+		}
+		add = prestige
+	}
+}
+
+conference_opposers_prestige_value = {
+	value = 0
+	every_country = {
+		limit = {
+			is_target_in_global_variable_list = {
+				name = current_congress_members_on_stance_3
+				target = this
+			}
+		}
+		add = prestige
+	}
+}
+
+conference_supporters_percentage_value = {
+	value = conference_supporters_prestige_value
+	divide = total_prestige_of_voting_conference_members
+}
+
+conference_neutral_percentage_value = {
+	value = conference_neutral_prestige_value
+	divide = total_prestige_of_voting_conference_members
+}
+
+conference_opposers_percentage_value = {
+	value = conference_opposers_prestige_value
+	divide = total_prestige_of_voting_conference_members
+}
+
+conference_power_balance_percentage_value = {
+	value = conference_supporters_percentage_value
+	subtract = conference_opposers_percentage_value
+}
+
+conference_sanctions_severity = {
```

Comment:
These severities should be script values at this point.

Reply:
We use the severity in multipliers, so we still need the number =/ 
So I'm a bit unsure of what you're asking for here